OpenRA.Mods.Common/Traits/MovementSounds.cs

```diff
@@ -0,0 +1,70 @@
+﻿// #region Copyright & License Information
+// /*
+//  * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+//  * This file is part of OpenRA, which is free software. It is made
+//  * available to you under the terms of the GNU General Public License
+//  * as published by the Free Software Foundation. For more information,
+//  * see COPYING.
+//  */
+// #endregion
+
+using System;
+using OpenRA.Traits;
+using OpenRA.Support;
```

Comment:
Don't forget to fix `[SA1210] Using directives must be sorted alphabetically by the namespaces.` when you do the rebase.